bp_top/test/tb/bp_tethered/Makefile.params

```diff
@@ -27,8 +24,6 @@ export USE_DRAMSIM2_LATENCY_P ?= 0
 export USE_MAX_LATENCY_P      ?= 1
 export USE_RANDOM_LATENCY_P   ?= 0
 
-export BP_SIM_CLK_PERIOD ?= 10
```

Comment:
How is the clock period now defined?

Reply:
Good point actually. Apparently the module needs to be clocked at the same period that's defined in the config file. So I think I've to add another clock region.

Reply:
Yeah, we'll need a CDC in the bp to dram adapter, and I think we'll also need this parameter (or define) for the core clock